community/tool-calls/spring-ai-alibaba-starter-tool-calling-crawler/src/main/java/com/alibaba/cloud/ai/toolcalling/crawler/CrawlerFirecrawlServiceImpl.java

```diff
@@ -33,7 +36,8 @@
  * https://jina.ai/reader/
  */
 
-public class CrawlerFirecrawlServiceImpl extends AbstractCrawlerService {
+public class CrawlerFirecrawlServiceImpl extends AbstractCrawlerService
```

Comment:
This ai crawler(jina & firecraw) just declares two but doesn't implement them, right?